es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number} ${street}`;
+
+    console.log('LearningFuze:', address);
+
+    const firstName = 'Uzair';
+    const lastName = 'Ashraf';
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log('Bio:', bio);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = `${fruit} ${item}`;
```

Comment:
A couple of these have had their spacing changed:

---

![image](https://user-images.githubusercontent.com/16143771/156108049-52ba3cd6-8d9e-4bdb-a0cb-03484330627d.png)

---

![image](https://user-images.githubusercontent.com/16143771/156108025-09c1a6a6-ba99-44fb-b4bf-15ec7cd198b6.png)